sway_libs/src/lib.sw

```diff
@@ -2,3 +2,10 @@ library sway_libs;
 
 dep merkle_proof/binary_merkle_proof;
 dep string/string;
+
+dep signed_integers/i8;
```

Comment:
These should be moved above `string` to be in alphabetical order

Reply:
Done

Reply:
Resolved https://github.com/FuelLabs/sway-libs/pull/30/commits/9196c79f8e7e138db517e72a563a721a41770aad